components/simplesat/sources/new-answer/new-answer.mjs

```diff
@@ -0,0 +1,59 @@
+import app from "../../simplesat.app.mjs";
+
+export default {
+  dedupe: "unique",
+  type: "source",
+  key: "simplesat-new-answer",
+  name: "New Answer",
+  description: "Emit new event when a new answer is created",
+  version: "0.0.1",
+  props: {
+    app,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      label: "Polling schedule",
+      description: "How often to poll the new answers",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  methods: {
+    _setLastAnswerId(id) {
+      this.db.set("lastAnswerId", id);
+    },
+    _getLastAnswerId() {
+      return this.db.get("lastAnswerId");
+    },
+  },
+  async run() {
+    let page = 1;
+    let firstId = null;
+    const lastId = this._getLastAnswerId();
+    do {
+      const res = await this.app.listAnswers(page);
```

Comment:
I think we can use `modified__gte` parameter and last answer's `modified` prop to reduce total page count and response time. Please let me know if it cannot be applied, then I will approve it because it looks good to me except this.

Reply:
Hi there. I would say that it would be the same. I would have to store one of these fields and apply the same logic since there is no filter for this endpoint